### PR TITLE
prep-css-box-model Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
Screenshot_20230111_121238

### CODE REVIEW

File: prep-css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding-top: 10px;
```
Comment (BrettAlbright) [inline prep-css-box-model/styles.css]: you want padding on all sides not just the top

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good so far, just make sure to update the padding to be on all four sides

**Comment by Nathan-Chon** [comment]
Screenshot_20230112_015536

"revised padding of code"

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**